### ISSUE TITLE
[FR]: Add 'env' to webpack_bundle

### ISSUE THREAD

**Issue by gregjacobs** [aspect-build/rules_webpack#63]
### What is the current behavior?

Currently, we can't pass environment variables to the Node.js process that runs webpack from the BUILD file config.

### Describe the feature

For my case in particular, I'm looking to provide `NODE_PATH` (to configure where node looks for node_modules), but I could imagine other users wanting to pass other configuration to the webpack process via environment variables.

Basically would just be another attribute of the `webpack_bundle` rule named `env`, just like most other rules currently accept.

### Fund our work

- [ ] Sponsor our open source work by donating a [feature bounty](https://opencollective.com/aspect-build/)

**Comment by gregmagolan** [comment]
Added in https://github.com/aspect-build/rules_webpack/pull/73.

Cutting a v.0.6.0 release that will include the new attribute.
